### ISSUE TITLE
No in-app messaging

### ISSUE THREAD

**Issue by pferhan** [pferhan/qBay#4]
Right now this service only offers contact information for the person selling but no easy way to follow up. In the future if this application were to become a reality, a more convenient in-app means of messaging would be necessary for quicker sales and smoother communication.

**Comment by Taylorlea43** [comment]
I agree, it would be much easier to contact someone if you could do it in the app, maybe then some information could be kept private.
